io_scene_godot/converters/multimesh.py

```diff
@@ -0,0 +1,177 @@
+import bpy
+import mathutils
+import math
+import re
+from ..structures import (
+    Array, NodeTemplate, InternalResource, Map, gamma_correct,fix_matrix)
+from .mesh import ArrayMeshResourceExporter
+
+def export_multimesh_node(escn_file, export_settings,
+                           obj, parent_gd_node):
+    """Export a blender object with parent_bone to a BoneAttachment"""
+    context = bpy.context
+    dg = context.evaluated_depsgraph_get()
+    ob = context.object.evaluated_get(dg)
+
+    ps = ob.particle_systems.active
+    if ps.settings.instance_collection and ps.settings.instance_collection.all_objects[0]:
+        instance_object=ps.settings.instance_collection.all_objects[0]
+    elif ps.settings.instance_object:
+        instance_object=ps.settings.instance_object
+
+    multimeshnode = NodeTemplate(re.sub('[^a-zA-Z0-9]','',ps.name) + 'MultiMesh',

```

Comment:
Names are already normalized here: https://github.com/godotengine/godot-blender-exporter/blob/564aa4f56cfe72be2d3c76893fe0380bb7548b8e/io_scene_godot/structures.py#L163-L164

If there's additional normalization needed it should probably be added there, unless it is specific to particles.